src/modules/CanHelp/CanHelp.ts

```diff
@@ -54,10 +54,10 @@ export namespace CanHelp {
             steps.Outlet.request,
             A.head,
             O.chain(req => Request.SupplyRequest(req, steps.Supply, steps.Summary)),
-            O.map(x => [x])
+            O.map(x => x)
           )
         ),
-        O.map(([helper, requests]) => ({ helper, requests })),
+        O.map(([helper, request]) => ({ helper, request })),
```

Comment:
I'm afraid we cannot merge it to master like that, as this will break the compatibility with existing backend.